csunplugged/config/settings/base.py

```diff
@@ -55,6 +55,7 @@
     'general.apps.GeneralConfig',
     'topics.apps.TopicsConfig',
     'resources.apps.ResourcesConfig',
+    'dev.apps.DevConfig'
```

Comment:
The application should only be available in `DEBUG` mode, not production.

To include it only for local development, add the following text to line 75 in `csunplugged/config/settings/local.py`:

```
LOCAL_APPS += ['dev.apps.DevConfig']
```

This is similar to lines 49 and 66 in the same file.